src/tensorlake/function_executor/proto/function_executor.proto

```diff
@@ -138,6 +159,54 @@ message RunTaskResponse {
     optional SerializedObject invocation_error_output = 10;
 }
 
+// TaskRequest contains the input data for task execution
+message TaskRequest {
+    optional SerializedObject function_input = 1;
+    optional SerializedObject function_init_value = 2;
+}
+
+// TaskResult contains the execution outcome and outputs of a task
+message TaskResult {
+    repeated SerializedObject function_outputs = 1;
+    // Edges that the function wants the invocation to be routed to.
+    repeated string next_functions = 2;
+    optional string stdout = 3;
```

Comment:
Aren't these getting moved into pre-signed urls? May be use SerializedObjects for these?

Reply:
That's Eugene's change.  For the current change, we want exactly the same fields that we currently get from run_task() / RunTaskRequest.